src/ray/raylet/node_manager.cc

```diff
@@ -519,6 +519,22 @@ ray::Status NodeManager::RegisterGcs() {
         [this] { local_object_manager_.FlushFreeObjects(); },
         RayConfig::instance().free_objects_period_milliseconds(),
         "NodeManager.deadline_timer.flush_free_objects");
+    periodical_runner_.RunFnPeriodically(
```

Comment:
Why is it better to spill here than the previous `HandleCreateObjectRequest`?

Reply:
I needed access to the number of bytes held by primary objects. LocalObjectManager has the ground truth.